dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -77,17 +77,21 @@ public void testArtifactsToPaths_removingDuplicates() throws RepositoryException
 
   @Test
   public void testBomToPaths_firstElementsAreBomMembers() throws RepositoryException {
+    
+    // that is, that the BOM members come before the transitive dependencies
```

Comment:
This comment just seems to be floating and incomplete

Reply:
It follows the method name.

Reply:
Statements that cross modalities can be very confusing. I'd much prefer a self-contained statement.

Reply:
Moved the comment. This is a weird test. I'm not 100% sure it's correct or that we need it. By Chesterton's fence I left it in, but it's worth another look.